website/docs/components/form/super-select/index.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+export default class Index extends Component {
+  OPTIONS = [
+    'Oregon (us-west-2)',
+    'N. Virginia (us-east-1)',
+    'Ireland (eu-west-1)',
+    'London (eu-west-2)',
+    'Frankfurt (eu-central-1)',
+  ];
+
+  GROUPED_OPTIONS = [
```

Comment:
Did a quick test and if an option has the same name then it's treated as the same even if under a different group. I can add a warning within the docs.

Reply:
Sync with @alex-ju on this. we discussed this briefly and he explained me that in that case the "shape" of the options in the group, instead of an array of strings, needs to be an array of objects, and the selected 
Probably this is documented in the PowerSelect APis so maybe we don't need to document it here? (or maybe instead we want to, because it's not so obvious?)